languages/README.md

```diff
@@ -0,0 +1,128 @@
+`languages/$(your-language-code)/$(corresponding-position)`. 
+For example:
+```
+cloudevents/spec.md should be translated as languages/zh-cn/cloudevents/spec.md
+cloudevents/adapters/github.md should be translated as languages/zh-cn/cloudevents/adapters/github.md
+*****NOTE: documents under languages directory should not be translated ***** 
+```
+If your language doesn't exist, add a new item to the [languages.md](languages.md) file, 
+which links to a newly-created directory named after the corresponding language code within 
+[language codes](https://en.wikipedia.org/wiki/Language_localisation#Language_tags_and_codes). 
+Then, add new translation to right position as described above.
+
+### Step 2. choose which document you'd like to translate
+A translator can know about which documents are not translated yet by reading a 
+"CE translation list".
+
+The list looks like this:
+
+![CloudEvents translation list](CE_translation_list.png)
+
+- version: choose from {`finished`, `working draft`}; a translator should always 
+  start translate from `finished` documents since `working draft` might be changed anytime.
+- status: choose from {`Ready to start`, `Started`, `PR reviewing`, `PR merged`}
+    - Ready to start: any document is ready to be translated stay in this state. 
+      A translator set status as `Started` once he/she decides to translate it.
+    - Started: indicating someone is translating this document. Translator and reviewer fields 
+      should be filled in once status is `Started`.
+    - PR reviewing: a translator finished the translation, and the work is being reviewed. 
+      The PR link field should be filled in once status is `PR reviewing`.
+    - PR merged: the translation is successfully merged in the spec. 
+      
+The file `languages/$(your-language-code)/translate.md` will hold the link of the 
+"CE translation list".
+
+If the file `languages/$(your-language-code)/translate.md` doesn't exist, 
+then you should:
+
+- Copy [translate-sample.md](translate-sample.md) to `languages/$(your-language-code)/translate.md`
+- Copy the "CE translation sample list" and create a new one for you. Then replace the sample sheet 
+  link with the new sheet link you just created.
+- Change the access right to "Anyone on the internet with this link can edit" so that every potential 
+  translator could directly choose a document to translate.
+- It doesn't matter which language you're using to write `languages/$(your-language-code)/translate.md`. 
+  However, it must contain the link of the "CE translation list" with access right as "everyone can edit".
+
+### Step 3. submit your translation
+A translator must submit translation via submitting a Pull Request. See details in how to 
+[submit a Pull Request](../community/CONTRIBUTING.md#suggesting-a-change).
+
+## Translation style guide
+- Only translate textual documents(generally md files). Other format documents(.json, .yaml, .g4) 
+  will not be translated.
```

Comment:
But they should presumably be copied into the directory without translation, right? That would be good to call out.

Reply:
Yep, non-textual documents will be copied to the right location, but not translated.